app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/screen/TeleconsultRecordEffectHandler.kt

```diff
@@ -23,9 +32,36 @@ class TeleconsultRecordEffectHandler @AssistedInject constructor(
         .addAction(GoBack::class.java, uiActions::goBackToPreviousScreen)
         .addConsumer(NavigateToTeleconsultSuccess::class.java, { uiActions.navigateToTeleconsultSuccessScreen(it.teleconsultRecordId) }, schedulersProvider.ui())
         .addTransformer(LoadTeleconsultRecordWithPrescribedDrugs::class.java, loadTeleconsultRecordWithPrescribedDrugs())
+        .addTransformer(CreateTeleconsultRecord::class.java, createTeleconsultRecord())
         .build()
   }
 
+  private fun createTeleconsultRecord(): ObservableTransformer<CreateTeleconsultRecord, TeleconsultRecordEvent> {
+    return ObservableTransformer { effects ->
+      effects
+          .observeOn(schedulersProvider.io())
+          .map { effect ->
```

Comment:
Since we aren't using the return value of `createTeleconsultRecordForMedicalOfficer` in the next operator, should this be a `.doOnNext` instead?